### DISCUSSION TITLE
Auto pretify ledger on save.

### DISCUSSION THREAD

**Discussion by avinashkanaujiya** [ananthakumaran/paisa#59] [Ideas]
As mentioned in title.

**Comment by ananthakumaran** [comment]
I don't know if everyone will like it. I will first add support for keyboard shortcuts. That will make it easier to save/format the file

**Comment by avinashkanaujiya** [comment]
Yes, that will work too.

**Reply by avinashkanaujiya** [reply]
Is it done?

**Reply by ananthakumaran** [reply]
Just added https://github.com/ananthakumaran/paisa/commit/cf99847de167ea701115b4e799f3f548d391b19a, will be available from next release.
